pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -174,3 +175,9 @@ type OperatorConfigurationUsers struct {
 
 //Duration shortens this frequently used name
 type Duration time.Duration
+
+type OperatorLogicalBackupConfiguration struct {
+	Schedule    string `json:"logical_backup_schedule,omitempty"`
+	DockerImage string `json:"logical_backup_docker_image,omitempty"`
+	S3Bucket    string `json:"logical_backup_s3_bucket,omitempty"`
```

Comment:
why so AWS-specific? Can it not be something like `logical_backup_target` everywhere?

Reply:
supporting other targets is outside the scope of this PR; such a change also has to be synced with the logical backup image env

Reply:
I would still claim, we could just remove s3 from here.

That is probably sufficient to leave any further work to the docker image.

Google, Digital Ocean, they probably all just take a "path" to a "bucket".